subsys/bluetooth/host/iso.c

```diff
@@ -141,6 +141,8 @@ int hci_le_reject_cis(uint16_t handle, uint8_t reason)
 	struct net_buf *buf;
 	int err;
 
+	BT_ERR("reason %d", reason);
```

Comment:
Seems unrelated. Move to separate commit or just remove

Reply:
Ah, yeah sorry, that was some debug statements I forgot to remove. Thanks!